### PR TITLE
Fix randomizer

### PR DESCRIPTION
**Pull request by maxnazaruk**
Since I've added the Tetraminos class in a different push session please take a view of that class separately.

### CODE REVIEW

File: src/main/java/com/example/demotetris/HelloApplication.java [hunk 1]
```diff
@@ -53,7 +50,7 @@ public class HelloApplication extends Application {
 
     public static final Random random = new Random();
 
-    private static List<Button[][]> elements = Arrays.asList(getSElemenet(), getLElement(), getFiveElement(), getPlaneElement(), getSquareElement(), getStickElemenet());
+    //private static List<Button[][]> elements = Arrays.asList(getSElemenet(), getLElement(), getFiveElement(), getPlaneElement(), getSquareElement(), getStickElemenet());
```
Comment (inazaruk) [inline src/main/java/com/example/demotetris/HelloApplication.java:53]: Prefer removing code to commenting it out. Very likely you'll never uncomment it and it'll just get stuck there.

File: src/main/java/com/example/demotetris/HelloApplication.java [hunk 2]
```diff
@@ -253,430 +250,6 @@ public static void main(String[] args) {
         launch(args);
     }
 
-    // TODO: in this method you are mixing rendering with "business logic". A better approach
```
Comment (inazaruk) [inline src/main/java/com/example/demotetris/HelloApplication.java:256]: This is a good chance, now we have much less boilerplate code in our "View" class.

**Review by inazaruk — APPROVED** [review]
Overall looks good!

File: src/main/java/com/example/demotetris/HelloApplication.java (hunk 1 above)
Comment (maxnazaruk) [reply src/main/java/com/example/demotetris/HelloApplication.java:53]: Agree

**Pull request merged**